EjercicioLaLiga Raul Pelayo

RaulPlayo opened the pull request.

RaulPlayo committed:
Create pelayoRaul

RaulPlayo committed:
Delete entregas/pelayoRaul

RaulPlayo committed:
Create README.md

RaulPlayo committed:
ejercicioLaLiga

mmasias commented:
Teniendo en cuenta las flechas que has agregado, que han creado varios ciclos, y por citar uno: 
- ¿Por qué llegamos al ciclo Equipo - Estadio - Partido y/o cómo se puede interpretar?

mmasias committed:
imagenes diversas

mmasias committed:
Ejercicios

RaulPlayo committed:
EjercicioSombra

RaulPlayo committed:
Cambios

RaulPlayo committed:
Update DosFocosUnaPersona.puml

RaulPlayo committed:
Update UnaPersonaDosFocos.puml

RaulPlayo committed:
ultimosCambios

RaulPlayo committed:
Merge branch 'main' of https://github.com/RaulPlayo/24-25-IdSw1

RaulPlayo committed:
Ejercicio003

mmasias committed:
Update ejercicio003.md

mmasias committed:
Update README.md

mmasias committed:
Update README.md

mmasias committed:
Update ejercicio003.md

mmasias committed:
Update ejercicio003.md

mmasias committed:
Añade reto de corrección para ejercicios

- Creado el archivo reto001.md con instrucciones para:
  - Corregir entregas anteriores según pautas de clase.
  - Documentar los ajustes de manera progresiva sobre los commits.
  - Enviar PR a la rama especificada una vez finalizado.

mmasias committed:
Añade tabla de retos con fechas en README

- Introducida una tabla con los detalles de los retos, incluyendo el enlace al reto 001, la fecha de entrega y la rama de entrega correspondiente.

RaulPlayo committed:
cambiosFinalizados

mmasias committed:
Create infoClientes.md

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Update infoClientes.md

mmasias committed:
Update infoEquipos.md

RaulPlayo committed:
Ejercicio10-10

RaulPlayo committed:
Update README.md

RaulPlayo committed:
diagramaEstados

RaulPlayo committed:
Merge branch 'main' of https://github.com/RaulPlayo/24-25-IdSw1

mmasias committed:
Update 1ra reunión

mmasias committed:
Update fechas

mmasias committed:
Refactoring++

RaulPlayo committed:
modelosDominio

mmasias committed:
APP

mmasias committed:
APP

RaulPlayo committed:
Merge branch 'mmasias:main' into main
